Formula/git-tools.rb

```diff
@@ -0,0 +1,39 @@
+class GitTools < Formula
+  include Language::Python::Shebang
+
+  desc "Assorted git-related scripts and tools"
+  homepage "https://github.com/MestreLion/git-tools"
+  url "https://github.com/MestreLion/git-tools/archive/v2022.12.tar.gz"
+  sha256 "b01e3b8de268ee07854e6cc66d78db6ed6cbc3e71dfeded8456138881bdf97f1"
+  license "GPL-3.0-only"
+  head "https://github.com/MestreLion/git-tools.git", branch: "main"
+
+  depends_on "python@3.11"
```

Comment:
Does this need `python@3.11` in particular, or does any `python3` interpreter do?

Reply:
Author of the tools, claims that `Python 3.1+` will do. I've tested it with Python 3.9.6 (the one is macOS 13) and it works fine.
```suggestion
  uses_from_macos "python", since: :catalina

  def install
    rewrite_shebang detected_python_shebang(use_python_from_path: true), "git-restore-mtime"

```
The above seems to work fine in macOS, however in Linux it seems to not rely on the systems python, but install `python@3.11`. Would that be ok?

Reply:
Yes that's fine.